.github/workflows/packaging.yml

```diff
@@ -0,0 +1,26 @@
+name: Packaging
+
+on:
+  release:
+    types: [released]
+
+
+jobs:
+  winget:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup winget-create
+        shell: powershell
+        run: |
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+      - name: Parse release version from github reference
+        run: Set-Item -PATH Env:\GITHUB_CLI_VERSION -Value ${{ github.ref }}.substring(1) 
+      - name: Update manifest
+        shell: powershell
+        run: |
+            wingetcreate.exe update --id GitHub,Inc.GitHubCLI --url https://github.com/cli/cli/releases/download/v$env:GITHUB_CLI_VERSION/gh_${env:GITHUB_CLI_VERSION}_windows_amd64.msi --version ${env:GITHUB_CLI_VERSION} --out .\manifest\
+      - name: Submit Pull Request to Winget Pkg
+        shell: powershell
+        run: wingetcreate.exe submit .\manifest\
```

Comment:
I think you need to pass in the `-t|--token` as well

```
> wingetcreate submit --help
Windows Manifest Creator v0.1.0.1

Copyright (c) Microsoft Corporation. All rights reserved.

The manifest creation command line utility generates manifest for submitting
apps to the Windows Package Manager repo.

USAGE:
Submit local manifest file to Windows Package Manager repo:
  wingetcreate.exe submit --token <GitHubPersonalAccessToken> <PathToManifest>

  -t, --token      GitHub personal access token used for direct submission to the Windows Package
                   Manager repo. If no token is provided, tool will prompt for GitHub login
                   credentials.
  Path (pos. 0)    Required. Path to a manifest file or directory containing the manifests that you
                   want to submit to the Windows Package Manager repo

More help can be found at: https://aka.ms/winget-create
Privacy statement: https://aka.ms/winget-create-privacy
```

Reply:
Added. I was gonna wait to ask @mislav for the token but I found one in `.github\workflows\release.yaml` so just used that  token.